### PR TITLE
[WIP] Add e2e github action

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    types:
+      - labeled
+      - opened
+      - synchronize
+      - closed
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ok-to-test') || github.event_name == 'closed' }}
```
Comment (elezar) [inline .github/workflows/e2e.yaml:31]: Does `|| closed` mean that this will trigger even if the PR is cancelled?

Comment (ArangoGutierrez) [reply .github/workflows/e2e.yaml:31]: Umm let me check the docs

Comment (ArangoGutierrez) [reply .github/workflows/e2e.yaml]: Removed

**Comment by ArangoGutierrez** [comment]
For Worklow dependency to work, the files need to be merged 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run

File: tests/e2e/holodeck.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: gh-action-5b6033b2
```
Comment (elezar) [inline tests/e2e/holodeck.yaml:4]: set this to `DUMMY` or `FILLED_BY_ACTION`

File: tests/e2e/holodeck.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  description: "Devel infra environment"
+spec:
+  provider: aws
+  auth:
+    keyName: cnt-ci
+    privateKey: "/Users/eduardoa/.ssh/cnt-ci.pem"
```
Comment (elezar) [inline tests/e2e/holodeck.yaml:10]: set this to `FILLED_BY_ACTION`

**Comment by ArangoGutierrez** [comment]
Going to recreate as a branched PR  (this is a Forked PR)

**Pull request closed without merging**